### DISCUSSION TITLE
Welcome to DeFooocus Discussions!

### DISCUSSION THREAD

**Discussion by ehristoforu** [ehristoforu/DeFooocus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by werzieq1** [comment]
\python_embeded\python.exe: can't open file 'E:\Defooocus\DeFooocus_win64_portable\DeFooocus\entry_with_update.py': [Errno 2] No such file or directory
Press any key to continue . . .


i just extracted and run as per instructions

**Reply by luckycharmsx** [reply]
The same error, anybody have an idea how to fix?

**Comment by departurechen** [comment]
I run into a bug, can anyone fix this for me ?

D:\defooocus\Fooocus>..\python_embeded\python.exe -m pip install -r requirements_versions.txt
Requirement already satisfied: torchsde==0.2.5 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 1)) (0.2.5)
Requirement already satisfied: einops==0.4.1 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 2)) (0.4.1)
Requirement already satisfied: transformers==4.30.2 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 3)) (4.30.2)
Requirement already satisfied: safetensors==0.3.1 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 4)) (0.3.1)
Requirement already satisfied: accelerate==0.21.0 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 5)) (0.21.0)
Requirement already satisfied: pyyaml==6.0 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 6)) (6.0)
Requirement already satisfied: Pillow==9.2.0 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 7)) (9.2.0)
Requirement already satisfied: scipy==1.9.3 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 8)) (1.9.3)
Collecting tqdm==4.65.0 (from -r requirements_versions.txt (line 9))
  Using cached tqdm-4.65.0-py3-none-any.whl.metadata (56 kB)
Requirement already satisfied: psutil==5.9.5 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 10)) (5.9.5)
Requirement already satisfied: pytorch_lightning==1.9.4 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 11)) (1.9.4)
Requirement already satisfied: omegaconf==2.2.3 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 12)) (2.2.3)
Requirement already satisfied: gradio==3.41.2 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 13)) (3.41.2)
Requirement already satisfied: pygit2==1.12.2 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 14)) (1.12.2)
Requirement already satisfied: opencv-contrib-python==4.8.0.74 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 15)) (4.8.0.74)
Collecting diffusers==0.25.1 (from -r requirements_versions.txt (line 16))
  Using cached diffusers-0.25.1-py3-none-any.whl.metadata (19 kB)
Requirement already satisfied: httpx==0.24.1 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 17)) (0.24.1)
Requirement already satisfied: onnxruntime==1.16.3 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 18)) (1.16.3)
Requirement already satisfied: timm==0.9.2 in d:\defooocus\python_embeded\lib\site-packages (from -r requirements_versions.txt (line 19)) (0.9.2)
Collecting translators==5.8.9 (from -r requirements_versions.txt (line 20))
  Using cached translators-5.8.9-py3-none-any.whl.metadata (68 kB)
Collecting rembg==2.0.53 (from -r requirements_versions.txt (line 21))
  Using cached rembg-2.0.53-py3-none-any.whl.metadata (14 kB)
Collecting groundingdino-py==0.4.0 (from -r requirements_versions.txt (line 22))
  Using cached groundingdino-py-0.4.0.tar.gz (82 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\admin\AppData\Local\Temp\pip-install-a2gzl7ww\groundingdino-py_bf46bddb9ea3476bbd6f2a5cf3622aea\setup.py", line 41, in <module>
          readme = readme_file.read()
      UnicodeDecodeError: 'gbk' codec can't decode byte 0xa4 in position 2878: illegal multibyte sequence
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

D:\defooocus\Fooocus>

**Comment by ego8898** [comment]
I've encountered an error, who can help me?

[notice] A new release of pip is available: 23.2.1 -> 24.0
[notice] To update, run: G:\DeFooocus_win64_portable\python_embeded\python.exe -m pip install --upgrade pip

CMD Failed requirements: install -r "requirements_versions.txt"
Total VRAM 8192 MB, total RAM 32612 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce RTX 3070 : native
VAE dtype: torch.bfloat16
Using split optimization for cross attention
Traceback (most recent call last):
  File "G:\DeFooocus_win64_portable\DeFooocus\entry_with_update.py", line 46, in <module>
    from launch import *
Refiner unloaded.
  File "G:\DeFooocus_win64_portable\DeFooocus\launch.py", line 128, in <module>
    from webui import *
  File "G:\DeFooocus_win64_portable\DeFooocus\webui.py", line 16, in <module>
    from modules.rembg import rembg_run
  File "G:\DeFooocus_win64_portable\DeFooocus\modules\rembg.py", line 1, in <module>
    from rembg import remove
ModuleNotFoundError: No module named 'rembg'
